optuna/storages/_rdb/storage.py

```diff
@@ -57,11 +58,52 @@
 
     models = _LazyImport("optuna.storages._rdb.models")
 
+
+_logger = optuna.logging.get_logger(__name__)
+
+
```

Comment:
Could you revert this change? It seems to be out-of-scope of this PR.

Reply:
It seems that I accidentally swapped this line with `_RDB_MAX_FLOAT`. Should I revert it?